src/main/kotlin/com/markskelton/OneDarkTheme.kt

```diff
@@ -3,18 +3,10 @@ package com.markskelton
 import com.intellij.openapi.project.DumbAware
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.startup.StartupActivity
-import com.markskelton.legacy.LegacyMigration
-import io.sentry.Sentry
 
 class OneDarkTheme : StartupActivity, DumbAware {
 
-  init {
-    Sentry.init("https://cb598170e51a44adbf0079abe2d79624@o403546.ingest.sentry.io/5267019?maxmessagelength=50000")
-  }
-
-
   override fun runActivity(project: Project) {
-    LegacyMigration.migrateIfNecessary()
     OneDarkThemeManager.registerStartup(project)
```

Comment:
Is this needed anymore?

Reply:
I thought I would keep it around to show the user update notifications :shrug:

Reply:
Sound good.